local-cli/generator-macos/templates/macos/Podfile

```diff
@@ -0,0 +1,40 @@
+# require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
+
+abstract_target 'Shared' do
+  # use_native_modules!
+
+  pod 'React', :path => "../node_modules/react-native-macos/"
+  pod 'React-Core', :path => "../node_modules/react-native-macos/React"
```

Comment:
I think the React-Core file is missing some source files for macOS.   When i run the init'd app I get runtime errors about StatusBar and other components.   It looks like the `React/Modules/RCTStatusBarManager.m` needs to be added to `React-Core-macOS` as it is in `React-Core-iOS`.   Its one of the things I had to fix in the static React.xcodeproj too for the initial PR for `react-native-macos-init`.

Reply:
Ya, I verified the fix locally.   Just remove the line in `React-Core.podspec` that is excluding `"Modules/RCTStatusBarManager.*"` for osx.   

There are probably a few more files that have to be un-excluded to re-align with the changes I made in the static XCodeproj.   I'll do that in a separate PR.

Reply:
Ah nice one 👍 I was going to ask about this one, but assumed it was currently a mobile only API. I’ll add this one in this PR and you can follow up with others.

Reply:
As per the comment on the guarded imports, we should perhaps see if we can have a CI task that does a build [of e.g. RNTester] that _only_ builds for iOS and then _only_ for macOS.

Reply:
Good idea.   I was going to update the CI to build RNTester using pods.   Its currently sliced by platform/flavor now but using the static xcodeproj's.   @HeyImChris FYI